public/less/components/footer.less

```diff
@@ -35,32 +35,39 @@
 
 i.logo#github {
     height: 32px !important;
-    padding: 0 0 0.75rem 0.75rem;
     margin-top: 18px;
     margin-left: -2px;
     float: right;
 }
 i.logo#twitter {
     height: 28px !important;
-    padding: 0 0 0.75rem 0.75rem;
     margin-top: 20px;
     float: right;
 }
 i.logo#fb {
     height: 28px !important;
-    padding: 0 0 0.75rem 0.75rem;
     margin-top: 20px;
     float: right;
 }
 i.logo#medium {
     height: 28px !important;
-    padding: 0 0 0.75rem 0.75rem;
     margin-top: 20px;
     float: right;
 }
 
-.logo {
-    font-size: 1.4em;
+
+@media screen and (min-width: 640px) {
```

Comment:
@redsn0w422  yeah that had more to do with the amount of things on the nav bar and when they would start behaving poorly, can you make the change to 425px and verify that it works (make sure i didnt do something wrong)?